source/configuration/kolla/nova.rst

```diff
@@ -28,6 +28,23 @@ PCI passthrough
      [    0.207523] DMAR-IR: IOAPIC id 15 under DRHD base  0xd37fc000 IOMMU 0
      [    0.207525] DMAR-IR: IOAPIC id 8 under DRHD base  0x9d7fc000 IOMMU 7
      [    0.207526] DMAR-IR: IOAPIC id 9 under DRHD base  0x9d7fc000 IOMMU 7
+     dragon@cpu1:~$ virt-host-validate
+     [...]
+     QEMU: Checking if IOMMU is enabled by kernel                               : PASS
+     [...]
+
+* disable nouveau in ``/etc/modprobe.d/blacklist.conf``
+
+  .. code-block:: console
+    
+     blacklist nouveau
+
```

Comment:
Run "sudo update-initramfs -u" after adding the blacklist entry and reboot the system afterwards.